packages/shared/src/graphql/squads.ts

```diff
@@ -555,3 +555,46 @@ export const getPublicSquadRequests = async (
 
   return res.requests;
 };
+
+export enum SourcePostModerationStatus {
+  Approved = 'approved',
+  Rejected = 'rejected',
+  Pending = 'pending',
+}
+
+export interface SourcePostModeration {
+  sourceId: string;
+  post: Post;
+  status: SourcePostModerationStatus;
+  postId: string;
+  moderatorId: string;
+  createdAt: Date;
+}
+
+export enum PostModerationReason {
+  OffTopic = 'OFF_TOPIC',
+  Violation = 'VIOLATION',
+  Promotional = 'PROMOTIONAL',
+  Duplicate = 'DUPLICATE',
+  LowQuality = 'LOW_QUALITY',
+  NSFW = 'NSFW',
+  Spam = 'SPAM',
+  Misinformation = 'MISINFORMATION',
+  Copyright = 'COPYRIGHT',
+  Other = 'OTHER',
+}
+
+export interface SquadPostRejectionProps {
+  postId: string;
+  reason: string;
+  note?: string;
+}
+
+export const squadApproveMutation = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _: string[],
+): Promise<void> => Promise.resolve();
+export const squadRejectMutation = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _: SquadPostRejectionProps,
+): Promise<void> => Promise.resolve();
```

Comment:
To be worked on once the API mutations are ready.

Reply:
Just add a Todo maybe so we don't forget

Reply:
Should have the todo now 🚀